### ISSUE TITLE
'Target' prop doesn't work

### ISSUE THREAD

**Issue by kalix127** [ZTL-UwU/shadcn-docs-nuxt#81]
### Environment

  System:
    OS: Linux 6.12 Manjaro Linux
    CPU: (16) x64 AMD Ryzen 7 4800H with Radeon Graphics
    Memory: 9.79 GB / 15.04 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 22.6.0 - ~/.nvm/versions/node/v22.6.0/bin/node
    npm: 10.8.3 - ~/.nvm/versions/node/v22.6.0/bin/npm
    pnpm: 9.15.1 - ~/.nvm/versions/node/v22.6.0/bin/pnpm
  Browsers:
    Brave Browser: 131.1.73.101
  npmPackages:
    nuxt: ^3.15.0 => 3.15.0 

### Describe the bug

The 'target' prop for the [TeamCard.vue](https://github.com/ZTL-UwU/shadcn-docs-nuxt/blob/c82b996ed2e28d2589dd48beb52e8e7a26fa5faf/components/content/TeamCard.vue) component doesn't work when passing the value '_blank'. 

Now im just gonna fork the repo, but in the next days i will try to fix this!

### Reproduction

By just using this snippet in any content page you will see that it doesn't open the link in a new tab.

```markdown
::team-card-group
  ::team-card
  ---
  avatar: https://www.github.com/ZTL-UwU.png
  name: Tony Zhang
  title: Author of shadcn-docs-nuxt
  links:
    - icon: lucide:github
      to: https://github.com/ZTL-UwU
      target: _blank
    - icon: lucide:twitter
      to: https://x.com/ZTL_UwU
      target: _blank
    - icon: lucide:coffee
      to: https://ko-fi.com/ztl_uwu
      target: _blank
  ---
  ::
::
```

**Comment by ZTL-UwU** [comment]
Fixed in [00e75b9](https://github.com/ZTL-UwU/shadcn-docs-nuxt/commit/00e75b9ae0efdb7a75c0602deb1d9093aa5d3ef5)

**Comment by kalix127** [comment]
in the next days i will open i think 2-3 more issues that i've found yesterday by using the template. (all minor issue)
